uiowa/uiowa
# Pre-card updates

The purpose of this PR is to extract some updates from #6054 that are not directly related to the card updates to hopefully make the process of reviewing and testing that PR a little easier.

A lot of these changes are minor syntactical changes handled automatically by PhpStorm.

# How to test
```
ddev blt ds --site sandbox.uiowa.edu && ddev drush @sandbox.local uli
```
- Review all code changes. 
- Pay attention to the code changes from double quotes and single quotes to ensure that is the only thing that changed. If any of them seem concerning, test the corresponding functionality.
- Horizontal menu `accessible-menu` library updates with no regressions. Check Tippie?
- Layout Builder styles patch to prevent duplicate classes from rendering. Find example of this on Sandbox and confirm that duplicate classes are no longer showing, but that classes are still showing.
- Article list block still defaults to 10 items per page.
- Blocks that show a placeholder in Layout Builder still have the `layout-builder-block--placeholder` added to them.
- When changing the media format style for card and image blocks, the underlying view mode changes.
- No regressions for narrow article feature.
- `inner--grid` class still gets removed from blocks where `inner--list` is selected.
- Headline and button classes still get set correctly on relevant blocks.
- Ensure that the card block button indicator functionality still renders correctly.

## Admissions
```
ddev blt ds --site admissions.uiowa.edu && ddev drush @admissions.local uli
```
- `admissions_core` module weight has changed. Spot check functionality to ensure there are no regressions.
- Compare AoS fields to ensure that they look the same: Link, Scholarship.
- Compare event series fields to ensure that they look the same: Button link, Summary.
- Compare major fields to ensure they look the same: CC checkpoints, Checkpoints.

## Now
```
ddev blt ds --site now.uiowa.edu && ddev drush @now.local uli
```
- `now_core` module weight has changed. Spot check functionality to ensure there are no regressions.

## Pharmacy
```
ddev blt ds --site pharmacy.uiowa.edu && ddev drush @pharmacy.local uli
```
- Compare person pages between the prod site and local to ensure no regressions.

## Grad Thesis Defense
```
ddev blt ds --site grad.uiowa.edu && ddev drush @grad.local uli
```
- Check thesis defense fields to ensure they look the same: Date, Location.
